src/bci_build/package/valkey.py

```diff
@@ -0,0 +1,45 @@
+"""Valkey container definition"""
+
+import textwrap
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import OsContainer
+from bci_build.package import OsVersion
+from bci_build.package import Replacement
+from bci_build.package import _build_tag_prefix
+
+VALKEY_CONTAINERS = [
+    ApplicationStackContainer(
+        name="valkey",
+        pretty_name="Persistent key-value database",
+        custom_description=(
+            "Valkey is a high-performance, open-source key-value "
+            "data store designed for a variety of workloads, "
+            "including caching, message queuing, and primary database use"
```

Comment:
From official docs:

Valkey is an open-source high-performance key/value datastore that supports a variety of workloads such as caching, message queues, and can act as a primary database

This sounds better and fixes a few typos.

Reply:
updated